Cargo.toml

```diff
@@ -30,7 +32,8 @@ themelio-stf = "0.11.4"
 melwallet-client= "0.6.7"
 num_cpus= "1.13.1"
 themelio-structs = "0.2.8"
-themelio-bootstrap = "0.6.1"
+# themelio-bootstrap = "0.6.1"
+themelio-bootstrap = { path = "../themelio-bootstrap" }
```

Comment:
@nullchinchilla any objections with bumping the `themelio-nodeprot` version up in `themelio-bootstrap`?

Reply:
No objections, but this should also bump `themelio-nodeprot` up in a semver-incompatible way (e.g. you need to change the `x` in `0.x.y`)

`themelio-bootstrap` as a whole is a bit weird --- ideally all it contains is some static data for bootstrapping, and other things depend on that rather than the other way around. It may be overdue for a refactor tbh